Vijay/Employee Admission Form/employee.html

```diff
@@ -103,7 +103,6 @@ <h2>WELCOME TO THE REGISTERATION PAGE</h2>
             var Qualification = document.getElementById("qua").value; 
             var Department = document.getElementById("dept").value;
             
-            
             let text = {Resume:Resume,Fname:Fname,Lname:Lname,DOB:DOB,Contact:Contact,Address:Address,
```

Comment:
Move this Script code to separate JS file and provide link in the HTML page.
Indent the code properly in JS file.

Reply:
Completed the given Reviews bro.